pyproject.toml

```diff
@@ -57,10 +61,28 @@ yapf = "*"
 
 [tool.poetry.extras]
 worker = [
+  "asn1crypto",
   "dfimagetools",
   "dfDewey",
+  "lief",
   "plaso",
+  "pyssdeep",
   "pyhindsight",
+  "py-tlsh",
+]
+server = [
+  "asn1crypto",
+  "lief",
+  "plaso",
+  "pyssdeep",
+  "py-tlsh",
+]
+api_server = [
```

Comment:
Drive by comment but i don't understand why you need to add these dependencies to the API server, or the server. These should only be required for the worker. Definitely do not want to add plaso as a dependency to the api server container here.

Reply:
Totally agree. This has surprised me too.
Have emailed you the docker logs in for server and api-server in case we do not include them.
Let's have a quick chat and hope we might be able to identify what the reason might be.

Reply:
Thanks heaps Juan for your help via email.
Have implemented your suggestion.
Hope it addresses your feedback.